manticore/ethereum/__init__.py

```diff
@@ -1320,14 +1320,13 @@ def generate_testcase(self, state, name, message='', only_if=None):
         For example, invariant: "balance" must not be 0. We can check if this can be violated and generate a
         testcase::
 
-            m.generate_testcase(state, 'balance', 'balance CAN be 0', only_if=balance == 0)
+            m.generate_testcase(state, 'balance CAN be 0', only_if=balance == 0, name='balance')
```

Comment:
let's actually just remove the `name='balance'` thing here. the 'balance' parameter was only there previously because some kind of `name` was required. now that's it's not, it's simpler to omit it.